data/planet.lua

```diff
@@ -576,7 +614,8 @@ local planets = {
     { "saturn", "土星", { "planet-discovery-jupiter", "space-platform-thruster" } },
     { "uranus", "天王星", { "planet-discovery-saturn", "space-platform-thruster" } },
     { "neptune", "海王星", { "planet-discovery-uranus", "space-platform-thruster" } },
-    { "pluto", "冥王星", { "planet-discovery-neptune", "space-platform-thruster" } }
+    { "pluto", "冥王星", { "planet-discovery-neptune", "space-platform-thruster" } },
+    { "asteroid-belt-outer", "柯伊伯带", { "planet-discovery-pluto", "space-platform-thruster" } }
```

Comment:
The Kuiper Belt was actually discovered after Pluto. You may want to change this anyway.